### ISSUE TITLE
remote_tmp should be configurable within group_vars and host_vars

### ISSUE THREAD

**Issue by tbar74235** [ansible/ansible#29019]
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME
remote_tmp var in ansible.cfg

##### ANSIBLE VERSION
```
ansible 2.2.2.0
  config file = /opt/jboss/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

##### CONFIGURATION
remote_tmp in ansible.cfg

##### OS / ENVIRONMENT
N/A

##### SUMMARY
The problem is that I am only able to configure the remote_tmp var within ansible.cfg.
But each host I like to administrate with ansible could have another path I wish to write the ansible tmp files to.

##### STEPS TO REPRODUCE
I should be able to configure the remote_tmp var within group_vars for example. Currently this will be ignored. Only the value of remote_tmp within ansible.cfg will be taken.

##### EXPECTED RESULTS
I would expect that if I would configure a remote_tmp property within group_vars or host_vars the remote_tmp property of ansible.cfg will be overwritten. So could define a special remote_tmp path for each host or host groups for example.

##### ACTUAL RESULTS
Currently only the remote_tmp property of ansible.cfg is the only option to change the remote_tmp path which every host you define within your ansible configuration must have.

**Comment by bcoca** [comment]
We already have plans to add this in the next version.

**Comment by tbar74235** [comment]
Would you specify the version which will contain this feature?

**Comment by bcoca** [comment]
it is on the 2.5 roadmap draft

**Comment by alikins** [comment]
See also https://github.com/ansible/ansible/issues/31022

**Comment by bcoca** [comment]
fixed via #31677
